# Lower minimum requirement on Cmake to 2.8.8
diff -urNad opencv-2.4.13.4-orig/CMakeLists.txt opencv-2.4.13.4/CMakeLists.txt
--- opencv-2.4.13.4-orig/CMakeLists.txt	2017-10-12 14:32:33.000000000 +0300
+++ opencv-2.4.13.4/CMakeLists.txt	2017-10-17 10:35:51.000769523 +0300
@@ -41,7 +41,7 @@
 elseif(IOS)
   cmake_minimum_required(VERSION 3.0)
 else()
-  cmake_minimum_required(VERSION 2.8.12.2)
+  cmake_minimum_required(VERSION 2.8.8)
 endif()
 
 if(POLICY CMP0026)
